config/jobs/kubernetes/sig-k8s-infra/trusted/OWNERS

```diff
@@ -2,8 +2,20 @@
 options:
   no_parent_owners: true
 
-reviewers:
-- sig-k8s-infra-oncall
-approvers:
-- sig-k8s-infra-oncall
-- test-infra-oncall
+filters:
+  ".*":
+    reviewers:
+    - sig-k8s-infra-oncall
+    approvers:
+    - sig-k8s-infra-oncall
+    - test-infra-oncall
+  # ContribEx related jobs that K8s infra owns (currently
+  # only the sig-contirbex-k8s-triage-robot.yaml job) should
+  # be reviewed by ContribEx leads since it could potentially
+  # constitute a workflow change:
+  # git.k8s.io/community/sig-contributor-experience/charter.md#cross-cutting-and-externally-facing-processes
+  "^sig-contribex-":
+    required_reviewers:
+      - sig-contributor-experience-leads
```

Comment:
@MadhavJivrajani I know I had lgtm'd https://github.com/kubernetes/test-infra/pull/29560 but changes like these should be only be lgtm'd by contribex TLs, not chairs.

https://github.com/kubernetes/community/blob/master/sig-contributor-experience/charter.md#additional-responsibilities-of-tech-leads

Can you update the alias in this PR? Apologies for the back and forth.

Reply:
Ah yes -- good catch. Let me change that, thanks.

Reply:
@nikhita done